### PR TITLE
Correcting ucp import statement to replect ucx-py PR#85 -- Also addin…

### PR DESCRIPTION
**Pull request by rjzamora**
…g simple error message for failed ucx import

### CODE REVIEW

File: distributed/comm/__init__.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ def _register_transports():
     try:
         from . import ucx
     except ImportError:
-        pass
+        print("UCX Import Error!")
```
Comment (mrocklin) [inline distributed/comm/__init__.py]: Two issues with this:

1.  We probably can't use print statements in library code.  If we wanted to do something here we would probably use the `logging` or `warnings` modules
2.  UCX won't be relevant for most Dask users (99% of Dask use today has nothing to do with RAPIDS or GPUs) so we'll need to optimize for that case.  Users without UCX installed shouldn't be troubled by a message like this.

**Comment by mrocklin** [comment]
I've manually merged this in.  Thanks @rjzamora !

**Comment by mrocklin** [comment]
@quasiben you can probably close this

**Comment by quasiben** [comment]
Thanks, sorry for the slow response

**Pull request closed without merging**